src/Templates/src/templates/maui-blazor/App.xaml.cs

```diff
@@ -1,17 +1,13 @@
-﻿using Microsoft.Maui;
-using Microsoft.Maui.Controls;
-using Microsoft.Maui.Controls.PlatformConfiguration.WindowsSpecific;
-using Application = Microsoft.Maui.Controls.Application;
+﻿using MauiApplication = Microsoft.Maui.Controls.Application;
```

Comment:
Why use `MauiApplication`?

Reply:
FYI @jamesmontemagno this one change was reverted because it wasn't clear it's necessary and it might create even more confusion with MAUI's 4 or 5 different "Application" types.

Reply:
sounds fine to me... why was it ever needed? Is application a conflict with the new usings at all though?